templates/faq.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% load static %}
+{% block content %}
+
+<!-- help needed -->
+<!-- needs to be handled from backend urls and views -->
+<h1>FAQ</h1>
+
+<p> Lorem ipsum dolor sit amet consectetur, adipisicing elit. Beatae, error cumque. Esse, obcaecati necessitatibus quasi
+    doloremque soluta cum voluptatum veritatis eius voluptas molestias a, magnam quam corporis. Voluptas, placeat
+    beatae.
+</p>
+<p>
+    Lorem ipsum dolor, sit amet consectetur adipisicing elit. Magnam fugiat fugit hic debitis eligendi unde eaque
+    facilis dignissimos commodi eos, odit nesciunt. Rerum quos suscipit quibusdam dolorum impedit temporibus sunt!
+</p>
+
+{% endblock %}
```

Comment:
@JimLynx 
You should always have a blank line (one only) at the end of each file.